third_party/license.txt

```diff
@@ -4744,3 +4744,20 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
+"""
+airflow
+"""
+Copyright 2018 Google Inc.
```

Comment:
This is clearly not the correct license text.

You should be able to find the right text in the airflow source code.

Reply:
It looks like the correct text is:

```
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
# http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

Reply:
What is the difference between what you have suggested and what I have typed?

Reply:
It doesn't match the text in the Airflow source code.

In particular, the Airflow text does not start with "Copyright 2018 Google Inc."

Reply:
I see now. Removing "Copyright 2018 Google Inc." will correct this. Please confirm.

Reply:
No, there are other differences, e.g. the formatting of the URL.

Just copy the Airflow text exactly.

Reply:
Done.